### ISSUE TITLE
Why only stop gradients of border_distribution's sample but not ones of scatter_density_distribution

### ISSUE THREAD

**Issue by SquareSon** [magdalena-wysocki/ultra-nerf#8]
Wonderful work! A great milestone of NeRF application in Ultrasound.

But I am confused about a small part of the render pipeline.

Why only stop gradients of border_distribution's sample but not ones of scatter_density_distribution

https://github.com/magdalena-wysocki/ultra-nerf/blob/d3155888f80d49f98f1ac6cdc26c1236876b0d40/run_ultra_nerf.py#L94
    border_indicator = tf.stop_gradient(border_distribution.sample(seed=0))
https://github.com/magdalena-wysocki/ultra-nerf/blob/d3155888f80d49f98f1ac6cdc26c1236876b0d40/run_ultra_nerf.py#L109
    scatterers_density = scatter_density_distibution.sample()
